deploy/general-deploy/README.md

```diff
@@ -1,30 +1,29 @@
-## Deploy Kubeturbo on Existing Kubernetes Cluster
+## Deploying Kubeturbo
 
-This guide is about how to deploy kubeturbo service in an existing Kubernetes cluster running in a private datacenter. It also applies to a local Kubernetes cluster. Kubeturbo is deployed as mirror on Master nodes.
+Once deployed, the Kubeturbo service enables you to give Turbonomic visibility into a Kubernetes cluster. This cluster can be located in either a private datacenter, or locally. Kubeturbo will be deployed as a mirror on Master nodes.
```

Comment:
the last part:
Kubeturbo will be deployed as a mirror pod on Master nodes.

I missed "pod" in my original doc.